Client/mods/deathmatch/logic/lua/CLuaManager.cpp

```diff
@@ -334,4 +334,5 @@ void CLuaManager::LoadCFunctions()
     CLuaWeaponDefs::LoadFunctions();
     CLuaWorldDefs::LoadFunctions();
     CLuaXMLDefs::LoadFunctions();
+    CLuaWorldDefs::LoadFunctions();
```

Comment:
Loading of CLuaWorldDefs functions occurs two lines above.
```suggestion
```

Reply:
Hmm. Seems like an unresolved merge conflict.